src/closedwindows/closedcard-directive.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.closedcard')
+        .directive('closedcard', [function() {
```

Comment:
`function()` and `() =>` behave slightly different in the way they handle `this`; we'd had some bugs using the old pattern before so to remain consistent they were all changed. Hopefully it'll minimise the change of unexpected bugs around this in the future.